Lets IPC's drink  +  Synthanol.

BeloneX opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request

As the title suggest IPC's can now drink from beakers and other container's this also adds new drinks made from synthanol.

## Why It's Good For The Game

Because you can't deconvert IPC cultists and there are tons of other, more troublesome instances of this being a bad thing.

This actually lets all species drink as long as they have heads, but the only instance of this being impactful is abductors, who can now down a few shots of whiskey before getting themselves killed.

"But Belone the Ayys will powergame with meth now and all that!", you say.

Well, they could've already used patches or syringes or divine intervention if they wanted to powergame with chems; this changes nothing.

They still can't eat solid food, just drink reagents from glasses, bottles and beakers.

----------------------------------------------------------------------------------------

Adds Synthanol, and sub drinks, so now IPCs can get drunk at the bar.

Synthanol can, and crushed can
![image](https://user-images.githubusercontent.com/47011211/95093118-b6175200-0728-11eb-8c02-d5bb4b987ebf.png)
![image](https://user-images.githubusercontent.com/47011211/95093123-b9124280-0728-11eb-9f52-e7e6fe5feb36.png)




Synthanol glass:
![image](https://user-images.githubusercontent.com/47011211/95093159-c16a7d80-0728-11eb-8664-5f54259b3299.png)

Space lube, plasma, fuel | 1:1:1 | Produces 3u

Robot tears:
![image](https://user-images.githubusercontent.com/47011211/95093527-2f16a980-0729-11eb-9049-ca2354331452.png)


Synthanol, oil, soda water | 1:1:1 | Produces 3u

Trinary:
![image](https://user-images.githubusercontent.com/47011211/95093563-350c8a80-0729-11eb-9f07-416e02f165f9.png)


Synthanol, lime juice, orange juice | 1:1:1 | Produces 3u

Servo:
![image](https://user-images.githubusercontent.com/47011211/95093210-d0e9c680-0728-11eb-8803-3625cde645a8.png)

Synthanol, cream, hot coco | 2:1:1 | Produces 4u

Uplink:
![image](https://user-images.githubusercontent.com/47011211/95093234-d6dfa780-0728-11eb-831d-1bd7e6ebb819.png)

Rum, vodka, tequila, whiskey, synthanol | 1:1:1:1:1 | Produces 5u

Synthsoda:
![image](https://user-images.githubusercontent.com/47011211/95093591-3b9b0200-0729-11eb-847b-2d56c2e48a3f.png)

Synthanol, cola | 1:1 | Produces 2u

Synthignon:
![image](https://user-images.githubusercontent.com/47011211/95093281-e4952d00-0728-11eb-904a-53799acc4803.png)

Synthanol, wine | 1:1 | Produces 2u

NOTE: Most of these drinks are animated ingame.

## Changelog
:cl:
add: Added Synthanol (And its sub drinks [Robot tears, Trinary, Servo, Uplink, Synthsoda, Synthignon]).
add: Added chemistry recipes for previously mentioned drinks (Wiki update coming soon).
tweak: Alcohol chem dispenser can now dispense synthanol.
imageadd: New sprites for these drinks.
fix: IPCs can now drink from glasses and cans.
tweak: IPCs can drink stuff now. Go get smashed, beep boop.
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

AnCopper committed:
synth drinks

ToasterBan commented:
ipc drinks are cool but ipcs can be deconverted using syringes

super12pl commented:
> ipc drinks are cool but ipcs can be deconverted using syringes

it takes more time though drinking is more convenient

TeomanTheGreat commented:
Ah yes a robot with no mouth can drink now

ghost commented:
> Ah yes a robot with no mouth can drink now

i don't have to explain shit, spessmagic

ss13-beebot committed:
Automatic changelog compile [ci skip]

Cenrus committed:
Add moth plush (#2454)

Co-authored-by: Crossedfall <ping@crossedfall.io>
Co-authored-by: ATH1909 <42606352+ATH1909@users.noreply.github.com>
Co-authored-by: ArcaneDefence <51932756+ArcaneDefence@users.noreply.github.com>

ss13-beebot committed:
Automatic changelog generation for PR #2454 [ci skip]

ss13-beebot committed:
Automatic changelog compile [ci skip]

ghost commented:
Incorrect grammar in your title I think, your title as of current says “Let us IPC’s drink”. If this is intended then drop the contraction, it makes it sound unnatural

ss13-beebot committed:
Automatic changelog compile [ci skip]

Chessus committed:
[PORT] Filter Blood Surgery + it's updates  (#2603)

* sds

* Adds blood filter to medical toolbelts and kit

* Add blood iflter to maps

* Removes from Autolathe

-Inline with no surgcial tools in autolathes

* Update backpack.dm

Removed from Deputy bag-No medical supplies present there

* Update blood_filter.dm

Holy shit, hours wasted.

* Changed chat output

* Airlock changes reverted

* Update DeltaStation2.dmm

missed a -26

* Forgot the ;

* Update PubbyStation.dmm

Last ;

ss13-beebot committed:
Automatic changelog generation for PR #2603 [ci skip]

ss13-beebot committed:
Automatic changelog compile [ci skip]

PowerfulBacon committed:
Update on_move.dm (#2659)

Co-authored-by: PowerfulBacon <>

ss13-beebot committed:
Automatic changelog compile [ci skip]

zeskorion committed:
some syringe injecting fixes (#2664)

* some syringe injecting fixes

* fix

ss13-beebot committed:
Automatic changelog generation for PR #2664 [ci skip]

ss13-beebot committed:
Automatic changelog compile [ci skip]

TeomanTheGreat committed:
New Hypospray Changes And Sound  (#2669)

* Hypospray Goes BRRRRRRRR

* Adds a space

ss13-beebot committed:
Automatic changelog generation for PR #2669 [ci skip]

ss13-beebot committed:
Automatic changelog compile [ci skip]

chloecat0 committed:
Some genetics fixes (#2648)

* fix some stuff

* more typos

* remove unnecessary code that prevents telekinesis

* remove nonexistant css class

* span fixes

ss13-beebot committed:
Automatic changelog generation for PR #2648 [ci skip]

ss13-beebot committed:
Automatic changelog compile [ci skip]

qwertyquerty committed:
Fixes 4 Random Runtimes (#2678)

* fix kinetic crusher exmaine null runtime

* fix gibspawner (passed in dna, should be living)

* fix remove eye control client null runtime

* fixes antag datum returning 0 instead of null on failure

ss13-beebot committed:
Automatic changelog generation for PR #2678 [ci skip]

ss13-beebot committed:
Automatic changelog compile [ci skip]

SomeAngryMiner committed:
The great resprite: Atomized edition (soup matter volume) (#2663)

* time to shove my greasy hand back inside bee's intestine

* (please dont shit your pants)

ss13-beebot committed:
Automatic changelog generation for PR #2663 [ci skip]

ss13-beebot committed:
Automatic changelog compile [ci skip]

St0rmC4st3r committed:
Lavaland science outpost (#2521)

* Adds an apc_nopower mapping helper

* Adds the science outpost shuttle

* Adds the lavaland science outpost and redundant areas for future works

* Adds a science shuttle dock to Pubby

* Adds a science shuttle dock to Meta

* Adds a science shuttle dock to Delta

* Adds a science shuttle dock to Box

* Why wasn't it in tgm in the first place?

* Ugly corners

* One less chair

* You can now restore the station-side consoles

* Now with a board!

* Working console on lavaland.

* Solves window and terminal testmerge issues.

* Makes box slightly prettier and removes redundant plastic flaps

* Moved crates

* Should resolve map conflicts

* fucky wucky

* Finally fixes stuff?!

* That one airlock now needs sci access

* Fixes regular bomb site camera being inaccessible.

ss13-beebot committed:
Automatic changelog generation for PR #2521 [ci skip]

ss13-beebot committed:
Automatic changelog compile [ci skip]

zeskorion committed:
nerfs cluwne rune (#2683)

* e

* oops

* eps

ss13-beebot committed:
Automatic changelog generation for PR #2683 [ci skip]

ss13-beebot committed:
Automatic changelog compile [ci skip]

Dennok committed:
Custom shuttle get name that you give it on construction. (#2686)

* shuttle get name of area

* Name for starting statinary port

ss13-beebot committed:
Automatic changelog generation for PR #2686 [ci skip]

ss13-beebot committed:
Automatic changelog compile [ci skip]

Cenrus committed:
initial (#2667)

zeskorion commented on code/modules/food_and_drinks/drinks/drinks.dm:
```diff
@@ -13,6 +13,7 @@
 	possible_transfer_amounts = list(5,10,15,20,25,30,50)
 	volume = 50
 	resistance_flags = NONE
+	you_drink_from_this = TRUE
```
can you perhaps check the open_container flag instead of using a snowflake var

zeskorion replied:
its not a var, it's a flag on reagent containers, and its not verbatim either

MNarath1 committed:
Fixes Circuits input length (#2661)

* Fix

* set it up to 6000

* set default to null instead of ""

* define a variable for the maxsize also increase size to 7000

* bump it up to 17500 as probably biggest possible circuit

* circuit wires are kinda broken so 20000 (why do wires have no limit?)

* Removes the null and calls directly max_length

* Reduced to 10000 cause yeah 20000 is a lot and might be too much

* Move define MAX_SIZE_CIRCUIT to say.dm for centralisation purpose

* Travis does not like this and honestly me neither so out it goes

* Set it up to 15000 (same as current music string cap)

ss13-beebot committed:
Automatic changelog generation for PR #2661 [ci skip]

ss13-beebot committed:
Automatic changelog compile [ci skip]

francinum committed:
[s] hotfix (#9951) (#2690)

* [s] hotfix

* log_admin

Co-authored-by: Maxim Nikitin <mnmaxim@gmail.com>

ike709 committed:
Compile your really important fix before opening the PR (#2691)

ike709 committed:
Dabs on the haters (#2692)

ss13-beebot committed:
Automatic changelog compile [ci skip]

PowerfulBacon committed:
Update browserOutput.dm (#2699)

ItsTisme committed:
Fix bug (#2698)

ss13-beebot committed:
Automatic changelog generation for PR #2698 [ci skip]

ss13-beebot committed:
Automatic changelog compile [ci skip]

VictorVow committed:
Adds THICKMATERIAL to various armoured exosuits and helmets (#2484)

* Adds THICKMATERIAL to many armoured suits and helmets

* Added THICKMATERIAL to armoured suits and helmets

Commit:
Adds new clothing for Security and the Captain. (#2621)

* new clothing

* whoops

* Update code/modules/clothing/suits/armor.dm

typo fix

Co-authored-by: Zamso <49600480+zeskorion@users.noreply.github.com>

* less armor

* fix

* lowercase

Co-authored-by: Zamso <49600480+zeskorion@users.noreply.github.com>

ss13-beebot committed:
Automatic changelog generation for PR #2484 [ci skip]

PowerfulBacon committed:
[BOUNTY] Atmos Bot (#2432)

* Atmosbot

* turf > turn

* Assembly, lag fixing

* Oops, forgot the sprite

* Final changes

* Fixes the toggle menus not working and runtime

* Fixes the making of them

* Fixes the scrubbing certain gasses control

* IT actually does something now

* booping whiring

* UI name

ss13-beebot committed:
Automatic changelog generation for PR #2621 [ci skip]

ss13-beebot committed:
Automatic changelog generation for PR #2432 [ci skip]

eeSPee committed:
Deputy Upgrades Passes (#2642)

* Deputy Upgrade

* deputy access cards

* Update deputy.dm

ss13-beebot committed:
Automatic changelog generation for PR #2642 [ci skip]

ss13-beebot committed:
Automatic changelog compile [ci skip]

VictorVow committed:
Shotgun rebalance (#2402)

* Riot and combat shotgun have a fire delay

* Better shotgun inhands

* Adds several new back slot icons for guns

* Shotgun ammo nerf

* Combat shotguns shoot faster, riots slower

* Reverted nonlethal shotgun ammo changes

* Reverted all non-cosmetic changes

ss13-beebot committed:
Automatic changelog generation for PR #2402 [ci skip]

ss13-beebot committed:
Automatic changelog compile [ci skip]

Crossedfall committed:
BYOND version bump for Travis (#2704)

* BYOND version bump for Travis

* Forgot this line

ss13-beebot committed:
Automatic changelog compile [ci skip]

yorii committed:
Update fancy.dm (#2703)

ss13-beebot committed:
Automatic changelog generation for PR #2703 [ci skip]

ss13-beebot committed:
Automatic changelog compile [ci skip]

AnCopper committed:
Merge branch 'IPCdrinks' of https://github.com/BeloneX/BeeStation-Hornet into IPCdrinks

ghost commented:
i accidentally rebased my master, meh it doesn't matter since it still complies, right?

zeskorion commented:
> i accidentally rebased my master, meh it doesn't matter since it still complies, right?

it does. gotta revert ro before you rebased then do it again

ghost commented:
broken
